src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -936,6 +936,136 @@ def test_containerapp_create_with_yaml(self, resource_group):
         user_identity_id = user_identity['id']
 
         # test managedEnvironmentId
+        containerapp_yaml_text = f"""
+            location: {TEST_LOCATION}
+            type: Microsoft.App/containerApps
+            tags:
+                tagname: value
+            properties:
+              managedEnvironmentId: {containerapp_env["id"]}
+              configuration:
+                activeRevisionsMode: Multiple
+                ingress:
+                  external: true
+                  allowInsecure: false
+                  targetPort: 80
+                  traffic:
+                    - latestRevision: true
+                      weight: 100
+                  transport: Auto
+              template:
+                revisionSuffix: myrevision
+                containers:
+                  - image: nginx
+                    name: nginx
+                    env:
+                      - name: HTTP_PORT
+                        value: 80
+                    command:
+                      - npm
+                      - start
+                    resources:
+                      cpu: 0.5
+                      memory: 1Gi
+                scale:
+                  minReplicas: 1
+                  maxReplicas: 3
+            identity:
+              type: UserAssigned
+              userAssignedIdentities:
+                {user_identity_id}: {{}}
+            """
+        containerapp_file_name = "containerapp.yml"
+
+        write_test_file(containerapp_file_name, containerapp_yaml_text)
+        self.cmd(f'containerapp create -n {app} -g {resource_group} --environment {env} --yaml {containerapp_file_name}')
+
+        self.cmd(f'containerapp show -g {resource_group} -n {app}', checks=[
+            JMESPathCheck("properties.provisioningState", "Succeeded"),
+            JMESPathCheck("properties.configuration.ingress.external", True),
+            JMESPathCheck("properties.environmentId", containerapp_env["id"]),
+            JMESPathCheck("properties.template.revisionSuffix", "myrevision"),
+            JMESPathCheck("properties.template.containers[0].name", "nginx"),
+            JMESPathCheck("properties.template.scale.minReplicas", 1),
+            JMESPathCheck("properties.template.scale.maxReplicas", 3)
+        ])
+
+        # test environmentId
+        containerapp_yaml_text = f"""
+                    location: {TEST_LOCATION}
+                    type: Microsoft.App/containerApps
+                    tags:
+                        tagname: value
+                    properties:
+                      environmentId: {containerapp_env["id"]}
+                      configuration:
+                        activeRevisionsMode: Multiple
+                        ingress:
+                          external: true
+                          allowInsecure: false
+                          targetPort: 80
```

Comment:
test with exposedPort

Reply:
exposedPort only support for TCP transport. Test exposedPort in test_containerapp_create_with_vnet_yaml.